pallets/registrar/src/lib.rs

```diff
@@ -1126,7 +1160,7 @@ where
         let signed_account =
             <frame_system::EnsureSigned<_> as EnsureOrigin<_>>::try_origin(o.clone())?;
 
-        if !Pallet::<T>::is_para_manager(para_id, &signed_account) {
+        if !Pallet::<T>::is_para_manager(para_id, signed_account.clone()) {
```

Comment:
You can avoid this clone by using `.as_ref()` inside that function to turn `Option<T>` into `Option<&T>`